### DISCUSSION TITLE
Prebuilt binaries

### DISCUSSION THREAD

**Discussion by GNUGradyn** [v1993/linuxmotehook2#16] [General]
Hello, I am trying to use this on my steam deck. I am having trouble getting it to compile. Can you provide prebuilt binaries please?

**Comment by v1993** [comment]
Fully portable prebuilt binaries on Linux tend to be tricky. Would you be satisfied with a PKGBUILD? I might even publish it to AUR if that's useful.

**Comment by GNUGradyn** [comment]
a pkgbuild would be fantastic, especially on the AUR

**Reply by v1993** [reply]
Alright, I've published [linuxmotehook2-git](https://aur.archlinux.org/packages/linuxmotehook2-git).

**Reply by v1993** [reply]
@GNUGradyn Did you have any success with AUR package?

**Reply by GNUGradyn** [reply]
Yes, much appreciated
